clio-transfer-model/src/main/scala/org/broadinstitute/clio/util/json/JsonSchema.scala

```diff
@@ -3,84 +3,53 @@ package org.broadinstitute.clio.util.json
 import java.net.URI
 
 import io.circe.Json
-import shapeless.{::, HList, HNil, LabelledGeneric, Lazy, Witness}
-import shapeless.labelled.FieldType
 
-import scala.reflect.runtime.universe.{Type, TypeTag, typeOf, typeTag}
+import scala.reflect.runtime.universe.{Type, typeOf}
 import java.time.OffsetDateTime
 import java.util.UUID
 
 import enumeratum.EnumEntry
-import org.broadinstitute.clio.util.generic.CirceEquivalentCamelCaseLexer
+import org.broadinstitute.clio.transfer.model.ClioIndex
+import org.broadinstitute.clio.util.generic.{CirceEquivalentCamelCaseLexer, FieldMapper}
 
 /**
-  * Generate JSON schema from case classes such as the QueryOutputs.
+  * A JSON schema for a key / metadata pair in Clio.
   *
-  * JsonSchema uses the same shapeless pattern as the
-  * [[org.broadinstitute.clio.util.generic.FieldMapper]]
-  *
-  * @tparam A is a case class suitable for Circe such as a QueryOutput
+  * @tparam CI The ClioIndex of the key / metadata pair.
   */
-trait JsonSchema[A] {
+class JsonSchema[CI <: ClioIndex](index: CI) {
```

Comment:
While I'm ripping stuff out... nobody actually uses the `get-schema` functionality. Can we get rid of it, too?